src/watch.js

```diff
@@ -427,29 +438,29 @@ class Watch {
      * Clears the change map.
      */
     const maybeReopenStream = function(err) {
-      if (isActive && !isPermanentError(err)) {
+      if (isActive && !self.isPermanentError(err)) {
         Firestore.log(
           'Watch.onSnapshot',
+          self._requestTag,
           'Stream ended, re-opening after retryable error: ',
           err
         );
         request.addTarget.resumeToken = resumeToken;
         changeMap.clear();
 
-        if (isResourceExhaustedError(err)) {
+        if (self.isResourceExhaustedError(err)) {
           self._backoff.resetToMax();
         }
 
         resetStream();
       } else {
-        Firestore.log('Watch.onSnapshot', 'Stream ended, sending error: ', err);
```

Comment:
This line is removed since it logs an error even when we explicitly close the stream ourselves. The actual error also gets logged by 'closeStream`.